src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig

```diff
@@ -701,6 +701,11 @@
 {% endblock birthday_widget %}
 
 {% block file_widget %}
+<style>
+  .custom-file-label:after {
+    content: "{{ "browse"|trans({}, 'Admin.Actions')  }}";
```

Comment:
Please write it _Browse_ instead, with a capital letter at the beginning, thank you.

Reply:
done !